okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/RemotingOkHttpCall.java

```diff
@@ -119,7 +119,9 @@ public void onFailure(Call _call, IOException exception) {
 
             @Override
             public void onResponse(Call _call, Response response) {
-                future.set(response);
+                if (!future.set(response)) {
+                   closeResponseBody(response);
+                }
```

Comment:
We already have this logic for retrofit clients, any way we can we avoid duplication?
https://github.com/palantir/conjure-java-runtime/blob/4.38.1/conjure-java-retrofit2-client/src/main/java/com/palantir/conjure/java/client/retrofit2/AsyncSerializableErrorCallAdapterFactory.java#L95-L98